ksp_plugin/flight_plan.cpp

```diff
@@ -166,7 +174,7 @@ std::unique_ptr<FlightPlan> FlightPlan::ReadFromMessage(
             AdaptiveStepSizeIntegrator<
                 Ephemeris<Barycentric>::NewtonianMotionEquation>::
                     ReadFromMessage(message.integrator()),
-            /*max_steps=*/1000,
+            /*max_steps=*/10000,
```

Comment:
Why? We'll deal with that properly in #938. If there's a good reason, it deserves a comment.